var/spack/repos/builtin/packages/morphio/package.py

```diff
@@ -14,6 +14,7 @@ class Morphio(CMakePackage):
     git      = "https://github.com/BlueBrain/MorphIO.git"
 
     version('develop', git=url, submodules=True)
+    version('3.1.1', tag='v3.1.1', submodules=True)
     version('3.0.2', tag='v3.0.2', submodules=True)
```

Comment:
Please update these versions in `py-morphio`, too :)

Reply:
my god ... I am so dumb

Reply:
Took me quite a while to notice this myself. After grepping for `depends_on.*py-morphio` not giving anything useful, that was the only place left to look.

Reply:
Yeah I have been grepping the py-morphio and all the possible dependencies containing it. I totally missed/forgot the normal morphio. Happens :)